docs/templates/reference-template.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Nouns describing your subject
+content_type: reference
+
+# Optional values. Uncomment any that apply to your document.
+
+# no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+# alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+# beta: true # Labels the page as beta; adds a banner to the top of the page.
+---
+
+Add an introduction paragraph here. Good introductions explain who this reference guide is for and what this guide will help the user accomplish.
+
+Reference material should not explain basic concepts. You can add links to explanation articles if needed.
+
+<!-- See https://documentation.divio.com/reference/ for more info about how to write a reference article -->
+<!-- See the following examples of reference documentation:
+* https://docs.konghq.com/gateway/latest/plugin-development/file-structure/
```

Comment:
I actually don't think that's the most exemplary reference topic. That one is a bit of a mix of types. For a pure reference, I'd use https://docs.konghq.com/gateway/latest/reference/router-expressions-language/